### DISCUSSION TITLE
Still loading dbt project, please try again later...

### DISCUSSION THREAD

**Discussion by WendyPeffercorn** [AltimateAI/vscode-dbt-power-user#354] [Q&A]
Hey everyone. My team has been using dbt cloud for a while, but we're making the switch to dbt core. A lot of us use VSCode and I managed to find this extension. It looks super great but I can't seem to get it working. 

I load up a model in the editor, but each time I try one of the dbt buttons (run, build, test, preview)... I just get met with "Still loading dbt project, please try again later... " I've tried searching around and can't specifically find this message on here or from searching google. I can open terminal and do dbt cli things. it just seems to be the buttons this addon places in the editor window.  

I've ran through the "how to set up the extension" portion. sql extensions configured for jinja, interpreter chosen, sqlfmt pipped and added to settings.json. What am I missing?

![image](https://user-images.githubusercontent.com/111669777/222574802-8904e0a9-661c-492b-9cb5-b06c349e1454.png)

**Comment by mihailradulov** [comment]
I'm facing the same issue both under windows and ubuntu 22.04. I did left vscode instance to try to load the project for more than 4 hours and nothing happened. Meanwhile there were no intensive processor load or memory usage.

**Comment by Vytautas-Lukosiunas** [comment]
The same on my side. Can't figure out problem, out of ideas and can't understand why package not detect dbt. 

Tried (windows 11):
- new venv
- reinstall dbt-bigquery
- cleaned vs code cache
- checked python

Any ideas welcome.

**Reply by anandgupta42** [reply]
Hi @Vytautas-Lukosiunas - Can you please file a bug with details here : https://github.com/innoverio/vscode-dbt-power-user/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml ?

We will have a look at the issue soon.

Thanks,
Anand

**Comment by Vytautas-Lukosiunas** [comment]
Hi,

Thanks for email.

I came back from conference and it works after your latest update!
Great and thanks for your support.


________________________________

**Comment by 8513421** [comment]
I have the same problem, I have the latest version of dbt power user... I'm out of ideas on how to fix it.

**Comment by anandgupta42** [comment]
@8513421 Can you please [file a bug with the details](https://github.com/innoverio/vscode-dbt-power-user/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml )?
